tests/cover/test_simple_numbers.py

```diff
@@ -67,6 +67,10 @@ def is_good(x):
     assert minimal(integers(min_value=boundary), lambda x: True) == boundary
 
 
+def test_minimizes_negative_integer_range_upwards():
```

Comment:
This has been added because of an area that was now showing up as uncovered (I think it must have been getting previously covered by a `@given` that is now not finding it due to engine changes)